src/profiler/profiler.c

```diff
@@ -170,10 +171,12 @@ static void profiler_write_header(FILE *file, char *script_name)
 	}
 	fprintf(file, "version: 1\ncreator: xdebug %s (PHP %s)\n", XDEBUG_VERSION, PHP_VERSION);
 	fprintf(file, "cmd: %s\npart: 1\npositions: line\n\n", script_name);
-	fprintf(file, "events: Time_(µs) Memory_(bytes)\n\n");
+	fprintf(file, "events: Time_(10ns) Memory_(bytes)\n\n");
 	fflush(file);
 }
 
+#define profiler_nanotime_scale(nanotime) ((unsigned long)((nanotime + 5) / 10))
```

Comment:
macros should be upper case, so: `PROFILER_NANOTIME_SCALE`, but as we're in the profiler file, and this doesn't leak, just use `NANOTIME_SCALE`.

Reply:
fixed